docs/source/whats_new.rst

```diff
@@ -1,3 +1,54 @@
+What's New in cartopy 0.15
+==========================
+
+:Release: 0.15.0
+:Date: 1st February 2017
+
+Features
+--------
+
+* The :class:`cartopy.crs.Mercator` class now allows a ``latitude_true_scale``
+  to be specified. 
+
+* A ``tiles`` url can now be passed directly to the
+  :class:`cartopy.img_tiles.GoogleTiles` class. 
```

Comment:
When I build the documentation, this isn't actually a link like :class:`cartopy.crs.Mercator` above is.

I noticed it also hasn't worked for the GoogleTiles reference here: https://github.com/SciTools/cartopy/blob/master/docs/source/whats_new.rst#id100 (and for a few other examples in the whats new)

I don't know why this is the case :confused: